### PR TITLE
Pokevision

### PR DESCRIPTION
**Pull request by iam4x**
screen shot 2016-07-26 at 09 42 15

Rewrite of #224 without performance issues. It refresh from pokevesion API every ~45seconds.

* [x] display pokémons from pokevision
* [x] display pokémons timeleft in human readable
* [ ] filter throughs pokémons
* [ ] button to force refresh pokémons positions (for instance after a teleport)
* [ ] button to toggle pokévision map (cc @h4v1nfun)
* [ ] click to go autopilot to pokémon
* [ ] credits pokevision on README

Thank's a lot to @Mardaneus86 for the original implementation <3

### CODE REVIEW

**Comment by Mardaneus86** [comment]
Nice rewrite, definitely have some more learning to do on React 😄.

Maybe we should also add an option to disable the pokevision overlay like @h4v1nfun suggested in the other PR, could be integrated in the filter as well.

**Comment by iam4x** [comment]
Sure, I've added it into the todo list 👍

File: src/views/map/index.js [hunk 1]
```diff
@@ -90,6 +92,17 @@ class Map extends Component {
             options={ () => this.mapOptions }
             onGoogleApiLoaded={ this.handleGoogleMapLoaded }
             yesIWantToUseGoogleMapApiInternals={ true }>
+            { /* display pokémon spots from pokévision */ }
+            { pokevision.pokemonSpots.map((p, idx) =>
+              <Pokemon
+                key={ idx }
```
Comment (Mardaneus86) [inline src/views/map/index.js]: I guess this should be 'p.id' instead of 'idx', otherwise it messes up when pokemon spots are removed from the array.

**Comment by caranicas** [comment]
Hey guys,  I love this app, but I really dislike pokevision. I find that it is often missing alot of the pokemon, and its locations are off or wrong. 

Ive been using [PokemonGo-Map](https://github.com/AHAAAAAAA/PokemonGo-Map), and just running it locally. I'm not sure what the lift would be to use this as a backend instead, but I think it would be a better user experience.

**Comment by fatherofawesome** [comment]
I would suggest integration with the latest PokemonGo-Map that most everyone is utilizing here: https://github.com/AHAAAAAAA/PokemonGo-Map

Other applications have integrated reading from it for it's purposes. See iOS app here: https://github.com/istornz/iPokeGo

**Comment by Mardaneus86** [comment]
Pokevision's service seems to be suspended for the moment (https://twitter.com/PokeVisionGo/status/759666743115513856). It's still not clear what Niantic and Nintendo specifically requested, but it's likely they don't want these kind of services around.

A PokemonGo-Map implementation seems to be the most reliable way to go from here on out I think. However, we would need a JS port of it to prevent a python requirement for this app.

**Comment by epilande** [comment]
@Mardaneus86 here's a node port, no python requirements https://github.com/Daplie/node-pokemap though I haven't actually tried it.

**Comment by iam4x** [comment]
Will see with `node-pokemap` but I'm on holidays right now.

**Pull request closed without merging**